01_fibonacci/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	for num := 1; num < 10; num++ {
+		fmt.Fprintln(out, fibinocci(num))
+	}
+}
+func fibinocci(x int) int {
+	if x <= 0 {
+		return 0
+	} else if x < 2 {
```

Comment:
Idiomatic Go omits the else when the if has a return:
```go
if x <= 0 {
	return 0
}
if x < 2 {
	return x
}
```
A more pertinent issue is that fibonacci sequences aren't defined for negative numbers, so you either should fail the call or leave it undefined (letting the `x < 2` clause handle them).